app/views/statistics/show.html.erb

```diff
@@ -3,31 +3,31 @@
 <div class="d-none d-md-flex justify-content-around text-center">
   <div class="jumbotron col-3">
     <h2 class="h4">
-      <%= t('statistics.dashboard.header.total_collections', count: @statistics_dashboard.collections.total) %>
+      <%= t('statistics.dashboard.header.total_collections', count: number_with_delimiter(@statistics_dashboard.collections.total)) %>
```

Comment:
I think either we'll want to abandon i18n pluralization or send two parameters: a count (as a number) for pluralization, and then the number_with_delimiter version?

Reply:
Got it. I've added the key `formatted_count` 🤷‍♂ . Its a little bit messier now.